src/cleric.cpp

```diff
@@ -164,7 +164,7 @@ bool cleric_ai_action(CharData *ch, CharData *victim) {
         case SPELL_SILENCE:
             for (victim = world[ch->in_room].people; victim; victim = next_victim) {
                 next_victim = victim->next_in_room;
-                if (IS_SPELLCASTER(victim) && FIGHTING(victim) == ch &&
+                if ((IS_SPELLCASTER_CLASS(victim) || IS_SPELLCASTER_RACE(victim)) && FIGHTING(victim) == ch &&
```

Comment:
Maybe make a `IS_SPELLCASTER(vict)` that does this.  Not requires, but might make it easier to read.  (Unless you want to specifically call out that this is both class and race.)

Reply:
Yep, was debating doing this anyway.  Just did.